File: server/core/model/token.go
package model

type Token struct {
	BaseModel
	TokenValue string `gorm:"column:token_value;size:150" json:"tokenValue,omitempty"`
	TokenDesc  string `gorm:"column:token_desc;size:200" json:"tokenDesc"`
}

func (Token) TableName() string { return "t_token" }

File: commom/src/main/java/com/javaoffers/nexus/common/model/Token.java
package com.javaoffers.nexus.common.model;

import com.javaoffers.brief.modelhelper.anno.BaseModel;
import com.javaoffers.brief.modelhelper.anno.BaseUnique;
import com.javaoffers.brief.modelhelper.anno.ColName;
import com.javaoffers.brief.modelhelper.anno.derive.flag.IsDel;
import lombok.Data;

import java.util.Date;

@BaseModel("t_token")
@Data
public class Token {
    @BaseUnique
    private Long id;
    private IsDel deleted;
    private Date createdAt;
    private Long createdBy;
    private Date updatedAt;
    private Long updatedBy;

    @ColName(value = "id", excludeColAll = true)
    private Long countId;

    private String tokenValue;
    private String tokenDesc;
}
